Example/ViewController.swift

```diff
@@ -65,6 +65,24 @@ class ViewController: UIViewController {
     }
 
     var alertController: UIAlertController!
+    // MARK: - Init
+    
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+        setAppDelegateProperty()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setAppDelegateProperty()
+    }
+    
+    private func setAppDelegateProperty() {
```

Comment:
This name focuses too much on “how” rather than “what” or “why”. Consider renaming it to `commonInit()`, which is a common pattern.